src/server.c

```diff
@@ -3922,6 +3922,23 @@ int processCommand(client *c) {
         return C_OK;
     }
 
+    /* If the server is paused, block the client until
+     * the pause has ended. Replicas are never paused. */
+    int is_readonly_command = (c->cmd->flags & CMD_READONLY) ||
+                              (c->cmd->proc == execCommand && !(c->mstate.cmd_inv_flags & CMD_READONLY));
```

Comment:
if we keep this var, i rather it'll move to the top (can be used for other checks some day)

Reply:
Ack